docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: "3"
+
+services:
+    db:
+        image: postgres
+        environment:
+            POSTGRES_DB: mariaquiteria
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: postgres
+        ports:
+            - "5432:5432"
+        volumes:
+            - ./db:/var/lib/postgresql
+
+    web:
+        build: .
+        volumes:
+            - .:/code
+        ports:
```

Comment:
O que acha de acrescentar um nginx + let's encrypt para produção? Se for assim podemos usar o docker-compose extends: https://docs.docker.com/compose/extends/ para colocar essa parte extra.

Reply:
Não precisamos implementar isso nesse PR, mas podemos abrir já um issue pra isso ;)